src/App.js

```diff
@@ -3,21 +3,31 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 import { useState, useEffect } from "react";
 
-// /*This is the app that will display the arrays of Object and the form with button*
 
-const useSemiPersistentState = () => {
+
+function App() {
   const [todoList, setTodoList] = useState(
     JSON.parse(localStorage.getItem("savedTodoList")) || []
```

Comment:
Since we added a `Promise` and data fetching in this lesson you can now initialize the `todoList` state as an empty array